### ISSUE TITLE
Keyboard navigation of the UI and accessibility

### ISSUE THREAD

**Issue by ColemanTom** [cylc/cylc-ui#2036]
### Problem

I have issues using mice and mice alternatives. I was trying to navigate the WUI with only a keyboard, but found I could not interact with any tasks directly. 

Using firefox

I could:

- select different workflows
- filter tasks
- filter task states
- pause/stop a workflow
- open the top accordion menu and interact with options with that menu
- scroll the view
- expand the try list (table view only)

I could not:

- open the workflow menu via the list on the left hand side
- access and interact with any tasks in the main area (tree view)

Whilst tabbing around, I also found there was an element which was selectable which served no purpose that I could find. If you tab to the tab selection (e.g. Table, Tree, etc), and then tab once more, it highlights a line that does not have the ability to interact (see the blue line in the image below).

![image](https://github.com/user-attachments/assets/f38dad9f-dae2-4109-9994-176b5dc6e1db)

I have not done extremely detailed testing of all functionality.

### Proposed Solution

Improve keyboard accessibiltiy for those who struggle with mouse usage, whether that be temporarily or permanently.

**Comment by oliver-sanders** [comment]
This is a duplicate of https://github.com/cylc/cylc-ui/issues/140

The blue line you mention *probably* comes from the Lumino library which unfortunately is beyond our control.

[Keyboard navigation](https://github.com/cylc/cylc-ui/issues/140) (along with the prerequisite of [row selection](https://github.com/cylc/cylc-ui/issues/434)) are intended features for the tree view (discussed in meetings, partially documented in the linked issue).

I'll translate these comments across to the original issue.
